chia/server/start_service.py

```diff
@@ -129,9 +130,44 @@ def __init__(
 
         self._on_connect_callback = on_connect_callback
         self._advertised_port = advertised_port
-        self._reconnect_tasks: Dict[PeerInfo, Optional[asyncio.Task[None]]] = {peer: None for peer in connect_peers}
+        self._connect_peers = connect_peers
+        self._connect_peers_task: Optional[asyncio.Task[None]] = None
         self.upnp: UPnP = UPnP()
 
+    async def _connect_peers_task_handler(self) -> None:
+        resolved_peers: Dict[UnresolvedPeerInfo, PeerInfo] = {}
```

Comment:
this dict seems to grow unbounded. is that OK?
I'm assuming `self._connect_peers` may have churn on peers. but perhaps not.

Reply:
It will only grow as big as `_connect_peers` which will not grow unbounded.